### ISSUE TITLE
SC1103: Missing wiki page for unknown shell directive rule

### ISSUE THREAD

**Issue by ParkerM** [koalaman/shellcheck#2616]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC1103
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
# shellcheck shell=blah
```

#### Here's what shellcheck currently says:
<pre><code>Line 1	<a href="https://www.shellcheck.net/wiki/SC1103">SC1103</a>: This shell type is unknown. Use e.g. sh or bash.</code></pre>

#### Here's what I wanted or expected to see:

The wiki page for https://www.shellcheck.net/wiki/SC1103 does not exist.

**Comment by brother** [comment]
The wiki is open for edits, please add the page.

**Comment by ParkerM** [comment]
Whoops, looks like you beat me to it, I updated it to reflect the directive rule rather than the shebang rule.

Would have went ahead and created the wiki entry but the guidance here seemed to suggest filing an issue instead: https://github.com/koalaman/shellcheck/wiki/Checks

**Comment by koalaman** [comment]
Thanks! The suggestion to file an issue is intended for people who come looking for information they don't have, since it would be hard for them to write it ^^

**Comment by TinCanTech** [comment]
Perhaps the default wiki `_footer` could point to a useful article.
